docs/team/fatin99.adoc

```diff
@@ -128,6 +129,7 @@ I made thorough *JUnit* tests for the `StudentCommand`, `StudentCommandParser`,
 * Removed the requirement for compulsory `Phone` and `Email` fields in `StudentAddCommand` (https://github.com/AY1920S2-CS2103T-W17-4/main/pull/146[#146]
 )
 * Managed the project by commenting on critical pull requests (various)
+** Managed release `v1.2` on GitHub
```

Comment:
Can help me delete this line please? don't need it anymore since I alrdy included it below haha